packages/gatsby-theme-api-docs/src/components/type/type.js

```diff
@@ -1,12 +1,10 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { FullWidthContainer } from '@commercetools-docs/gatsby-theme-docs';
-import filterOutApiTypeSubtypes from '../../utils/filter-out-api-subtypes';
 import { generateTypeURN } from '../../utils/ctp-urn';
 import { useApiTypes } from '../../hooks/use-api-types';
 import reportError from '../../utils/report-error';
 import Children from './children';
```

Comment:
I don't see a point why we still need the "children" abstraction - can the "children" component be merged directly into "type?